WordPress/Classes/ViewRelated/Me/My Profile/Change Username/ChangeUsernameViewController.swift

```diff
@@ -135,44 +145,50 @@ private extension ChangeUsernameViewController {
                                                 preferredStyle: .alert)
         alertController.addAttributeMessage(String(format: Constants.Alert.message, viewModel.selectedUsername),
                                             highlighted: viewModel.selectedUsername)
-        alertController.addCancelActionWithTitle(Constants.Alert.cancel, handler: { [weak alertController] _ in
-            if let textField = alertController?.textFields?.first {
-                NotificationCenter.default.removeObserver(textField, name: UITextField.textDidChangeNotification, object: nil)
```

Comment:
I believe the intention is removing the observer that's added when creating this alert. But this line doesn't actually do that: `textField` is not registered as an observer, not by this view controller at least.